homeassistant/components/homematicip_cloud/cover.py

```diff
@@ -48,22 +48,22 @@ def current_cover_position(self):
         """Move the cover to a specific position."""
         position = kwargs[ATTR_POSITION]
         level = position / 100.0
-        await self._device.set_shutter_level(level)
+        await self._device.set_shutter_level(1-level)
```

Comment:
Move this adjustment into the assignment of `level` itself:  `level = 1 - position / 100.0`